rpcs3/rpcs3qt/register_editor_dialog.cpp

```diff
@@ -187,10 +189,11 @@ void register_editor_dialog::updateRegister(int reg)
 			const u32 reg_index = reg % 128;
 			str = fmt::format("%016llx%016llx", spu.gpr[reg_index]._u64[1], spu.gpr[reg_index]._u64[0]);
 		}
-		else if (reg == MFC_PEVENTS) str = fmt::format("%08x", +spu.ch_event_stat);
-		else if (reg == MFC_EVENTS_MASK) str = fmt::format("%08x", +spu.ch_event_mask);
+		else if (reg == MFC_PEVENTS) str = fmt::format("%08x", +spu.ch_events.load().events);
+		else if (reg == MFC_EVENTS_MASK) str = fmt::format("%08x", +spu.ch_events.load().mask);
+		else if (reg == MFC_EVENTS_COUNT) str = fmt::format("%u", +spu.ch_events.load().count);
 		else if (reg == MFC_TAG_MASK) str = fmt::format("%08x", spu.ch_tag_mask);
-		else if (reg == SPU_SRR0) str = fmt::format("%08x", spu.srr0);
+		else if (reg == SPU_SRR0) str = fmt::format("%05x", spu.srr0);
```

Comment:
Same question here, I would have thought more uniform formatting is easier on the eyes regardless of underlying data structures.

Reply:
srr0's value can span on 5 digits at max (0x3ffff), this is consistent with the SPU debugger as well.

Reply:
Well, I can just more error checking instead.

Reply:
If its already being clamped to 5 digits in other views then its fine, although I personally prefer zero padding to balance out visuals. It's a subjective thing anyway.